Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -823,7 +823,84 @@ extension PaymentSheetUITest {
         try loginAndPay()
     }
 
-    private func loginAndPay() throws {
+    func testLinkAddCard_CollectingBillingDetails() throws {
```

Comment:
Because cards are so important, seems like we should add a similar test with link off?

Reply:
That's the existing behavior and it is already covered by the link tests